src/metpy/units.py

```diff
@@ -44,10 +44,15 @@ def _fix_udunits_powers(string):
     return _UDUNIT_POWER.sub('**', string)
 
 
+def _fix_udunits_div(string):
+    return 's**-1' if re.fullmatch('/s', string) is not None else string
```

Comment:
I don't think you even need a regex here, unless you wanted to expand this beyond just seconds (which I'd be opposed to in the absence of actual examples):
```suggestion
    return 's**-1' if string == '/s' else string
```

Reply:
Nice! Yep, appears to work. I'll also update the comment a little to indicate the further corrections being done.